src/engraving/libmscore/chord.cpp

```diff
@@ -1830,6 +1836,15 @@ void Chord::layoutStem()
 {
     // we should calculate default stem length for this chord even if it doesn't have a stem
     // because this length is used for tremolos or other things that attach to where the stem WOULD be
```

Comment:
Shouldn't this comment stay together with the line `_defaultStemLength = calcDefaultStemLength();`?

Reply:
yep, good catch, fixed